schema/table.go

```diff
@@ -199,9 +200,26 @@ func (t Table) Resolve(ctx context.Context, meta ClientMeta, parent *Resource, r
 			continue
 		}
 		for i := range objects {
-			summary.Merge(t.resolveObject(ctx, meta, parent, objects[i], resolvedResources))
+			i := i
+
+			// right now we support only concurrency
```

Comment:
I don't quite understand what this comment is trying to convey; it looks like we're supporting both?

Reply:
Yeah it adds unnecessary confusion. the extra `nil` in line 273 should be enough.

Reply:
I updated the comment (prob got distracted in the middle of the sentence)